images/proftpd/entrypoint.sh

```diff
@@ -18,12 +18,35 @@ if [ -e /run/secrets/$FTPUSER_PASSWORD_SECRET ] && ! id -u "$FTPUSER_NAME"; then
     | chpasswd -e
 fi
 
+if [ "$SFTP_ENABLE" = "on" ]; then
+  mkdir /etc/ssh
+  ssh-keygen -m PEM -f /etc/ssh/ssh_host_rsa_key -N '' -t rsa -b 2048
+  ssh-keygen -m PEM -f /etc/ssh/ssh_host_dsa_key -N '' -t dsa -b 1024
+  ssh-keygen -m PEM -f /etc/ssh/ssh_host_ecdsa_key -N '' -t ecdsa -b 521
+
+  sed -i -e "/^Port/s/^/#/" /etc/proftpd/proftpd.conf
+  
+  sed -i \
+      -e "s/^#\(  SFTPEngine on\)/\1/" \
+      -e "s/^#\(  Port 2222.*\)/  Port $SFTP_PORT/" \
+      -e "s/^#\(  SFTPCompression delayed\)/\1/" \
+      -e "s/^#\(  SFTPHostKey\)/\1/" \
+      /etc/proftpd/conf.d/sftp.conf
+fi
+
+if [ "$ANONYMOUS_DISABLE" = "on" ]; then
+  sed -i '/<Anonymous/,/<\/Anonymous>/d' /etc/proftpd/proftpd.conf
+else
+  sed -i \
+    -e "s:{{ ANONYMOUS_DISABLE }}:$ANONYMOUS_DISABLE:" \
+    -e "s:{{ ANON_UPLOAD_ENABLE }}:$ANON_UPLOAD_ENABLE:" \
+    /etc/proftpd/proftpd.conf
+fi
+
```

Comment:
This section which modifies the `<Anonymous>` block in proftpd.conf should be mentioned in PR description, it's unrelated to the SFTP updates. LGTM if tested fully.